DQM/L1TMonitorClient/python/L1TObjectsTimingClient_cff.py

```diff
@@ -0,0 +1,303 @@
+import FWCore.ParameterSet.Config as cms
+from DQMServices.Core.DQMEDHarvester import DQMEDHarvester
+
+# directory path shortening
+l1tobjectstimingDqmDir = 'L1T/L1TObjects/'
+
+# L1TObjects Timing Ratio Plots
+l1tObjectsRatioPlots = DQMEDHarvester("DQMGenericClient",
+    subDirs = cms.untracked.vstring(l1tobjectstimingDqmDir),
+    efficiency = cms.vstring(
+       "Ratio_L1TMuons_BX_minus2 'Ratio for L1TMuons for BX=-2' L1TMuon/timing/muons_eta_phi_bx_minus2 denominator",
+       "Ratio_L1TMuons_BX_minus1 'Ratio for L1TMuons for BX=-1' L1TMuon/timing/muons_eta_phi_bx_minus1 denominator",
+       "Ratio_L1TMuons_BX_0 'Ratio for L1TMuons for BX=0' L1TMuon/timing/muons_eta_phi_bx_0 denominator",
+       "Ratio_L1TMuons_BX_plus1 'Ratio for L1TMuons for BX=+1' L1TMuon/timing/muons_eta_phi_bx_plus1 denominator",
+       "Ratio_L1TMuons_BX_plus2 'Ratio for L1TMuons for BX=+2' L1TMuon/timing/muons_eta_phi_bx_plus2 denominator",
+       "Ratio_L1TJet_BX_minus2 'Ratio for L1TJet for BX=-2' L1TJet/timing/jet_eta_phi_bx_minus2 denominator",
```

Comment:
You still need separate denominators for the muons, jet, taus, egamma, and etsums.